### PR TITLE
Oberon NET SDK

### PR DESCRIPTION
**Pull request by tmarkovski**
Do not merge until #186 is merged

### CODE REVIEW

File: include/okapi.h [hunk 1]
```diff
@@ -47,6 +47,18 @@ int32_t ldproofs_verify_proof(struct ByteBuffer request,
...
+                            struct ExternError *err);
+
+int32_t oberon_create_proof(struct ByteBuffer request,
+                            struct ByteBuffer *response,
+                            struct ExternError *err);
+
+int32_t oberon_verify_proof(struct ByteBuffer request,
+                            struct ByteBuffer *response,
+                            struct ExternError *err);
+
```
Comment (fundthmcalculus) [inline include/okapi.h:61]: NO ACTION: I will add these to python/golang/ruby/java/swift/etc whenever you're ready.

File: native/src/tests/oberon.rs [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        sk: vec![
+            16, 133, 126, 11, 192, 153, 22, 14, 53, 214, 99, 40, 66, 194, 96, 30, 19, 86, 137, 107,
+            150, 49, 104, 202, 209, 80, 128, 182, 15, 154, 34, 57, 100, 51, 175, 108, 12, 56, 6,
+            76, 46, 173, 247, 255, 184, 165, 228, 127, 145, 65, 171, 195, 44, 164, 3, 16, 132, 43,
+            108, 82, 63, 136, 116, 3, 93, 1, 226, 152, 197, 152, 61, 212, 185, 32, 195, 211, 37,
+            206, 242, 31, 72, 79, 83, 71, 197, 102, 202, 129, 95, 19, 105, 34, 22, 46, 124, 94
+        ],
+        data: Vec::new(),
+        blinding: Vec::new(),
+    };
```
Comment (fundthmcalculus) [inline native/src/tests/oberon.rs:26]: Could we use this as an example .NET test?

**Review by fundthmcalculus — CHANGES_REQUESTED** [review]
See comments. `NOACTION` means it's a question for my clarification/posterity, not a requested change.

File: dotnet/Tests/Okapi.Tests/OberonTests.cs [hunk 3]
```diff
@@ -0,0 +1,4 @@
+﻿namespace Okapi.Tests
+{
+    public class OberonTests { }
```
[the remark this answers is not in the document]
Comment (tmarkovski) [reply dotnet/Tests/Okapi.Tests/OberonTests.cs:3]: Will add with this PR

File: native/src/tests/oberon.rs (hunk 2 above)
Comment (tmarkovski) [reply native/src/tests/oberon.rs:26]: Yes, but I'll add tests in the net project that use didkey instead.

**Pull request merged**